index.html

```diff
@@ -705,7 +705,9 @@ <h2 id="AudioContext">
           <dd>
             <p>
               This is a time in seconds which starts at zero when the context
-              is created and increases in real-time. All scheduled times are
+              is created and increases in real-time. At any given point, its value
+              represents the starting time of the next block of audio to be processed
```

Comment:
Joe Berkovitz writes:

>> @@ -705,7 +705,9 @@ <h2 id="AudioContext">
>>            <dd>
>>              <p>
>>                This is a time in seconds which starts at zero when the context
>> -              is created and increases in real-time. All scheduled times are
>> +              is created and increases in real-time. At any given point, its value
>> +              represents the starting time of the next block of audio to be processed

Perhaps "indicates the time immediately after the last block of
audio processed".

It means almost the same thing, but may differ if there is a
block currently being processed, and avoids any hint of
implication that using the value of currentTime will guarantee
scheduling at precisely that time.